src/component/entity-form.js

```diff
@@ -0,0 +1,28 @@
+import {customElement, bindable} from 'aurelia-framework';
+import {resolvedView} from 'aurelia-view-manager';
+
+@resolvedView('spoonx/form', 'entity-form')
+@customElement('entity-form')
+export class EntityForm {
+
+  @bindable entity;
+
+  @bindable behavior;
+
+  @bindable skip = [];
+
+  attached() {
+    let types = this.entity.getMeta().metadata.types;
```

Comment:
`types` is an object, that's why I used map and populated `elements`. I would need a valueConverter to loop through `types` in the view. Which will basically be the same as I do now.